app/mailers/invitation_mailer.rb

```diff
@@ -1,7 +1,9 @@
 class InvitationMailer < ApplicationMailer
-  def demo_invite(users)
-    users.each do |user|
-      mail(to: user.email, subject: 'demo')
-    end
+  def demo_invite
+    @user = params[:user]
+    work_group = params[:work_group]
```

Comment:
because I don't use the variable outside of this method :) 
you can see that all the instance variables are used in the mailer view, but not work_group, hence why there's no need to assign it as an instance variable, but just a local one is fine